mirage/scenarios/default.js

```diff
@@ -1,80 +1,80 @@
 export default function(server) {
-  server.createList('vendor', 5, {
+  server.createList('vendor', 5, 'withPackagesAndTitles', {
     packagesTotal: () => Math.floor(Math.random() * 10) + 1
   });
 
-  function createVendor(name, packages = []) {
```

Comment:
Commented out this test data for now, since some of these title counts are pretty substantial. It's pretty noticeably slowing down mirage.